__tests__/subscriptions.test.js

```diff
@@ -204,6 +207,59 @@ describe('subscription methods', () => {
 		});
 	});
 
+	describe('reschedulePayment', () => {
+		const path = '/subscription/payments_reschedule';
+		const expectedBody = Object.assign(
+			{
+				payment_id: PAYMENT_ID,
+				date: NEW_PAYMENT_DATE,
+			},
+			EXPECTED_BODY
+		);
+
+		it('resolves on successful request', () => {
+			// https://developer.paddle.com/api-reference/fe93f28aa7f7e-reschedule-payment
+			const body = {
+				success: true,
+				response: [
+					{
+						success: true,
+					},
+				],
+			};
+
+			const scope = nock()
+				.post(path, expectedBody)
+				.reply(200, body);
+
+			return instance
+				.reschedulePayment(PAYMENT_ID, NEW_PAYMENT_DATE)
+				.then(response => {
+					expect(response).toEqual(body.response);
+					expect(scope.isDone()).toBeTruthy();
+				});
+		});
+
+		it('rejects on invalid date format', () => {
+			expect(() =>
+				instance.reschedulePayment(PAYMENT_ID, NEW_PAYMENT_DATE_INVALID)
```

Comment:
Please inline the value of `NEW_PAYMENT_DATE_INVALID` since it's not being used anywhere else.

Reply:
Good idea, makes sense.